src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package kitchenpos.application;
+
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+@DisplayName("메뉴 그룹 서비스에 관련한 기능")
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+    @Mock
+    private MenuGroupDao menuGroupDao;
+    @InjectMocks
+    private MenuGroupService menuGroupService;
+
+    private MenuGroup 세트메뉴;
+
+    @BeforeEach
+    void beforeEach() {
+        세트메뉴 = new MenuGroup();
+        세트메뉴.setId(1L);
+        세트메뉴.setName("세트메뉴");
+    }
+
+    @DisplayName("`메뉴 그룹`을 생성한다.")
+    @Test
+    void createMenuGroup() {
+        // Given
+        given(menuGroupDao.save(any())).willReturn(세트메뉴);
+        // When
+        MenuGroup actual = menuGroupService.create(세트메뉴);
+        // Then
+        assertAll(
+                () -> assertNotNull(actual),
+                () -> assertEquals(세트메뉴.getId(), actual.getId()),
+                () -> assertEquals(세트메뉴.getName(), actual.getName())
+        );
```

Comment:
코드 작성에 있어서 크게 중요한 부분은 아니지만 `Given`, `When`, `Then` 구문 별 라인을 추가해 분리해주시면 가독성이 더 좋을 것 같습니다!

```suggestion
        // Given
        given(menuGroupDao.save(any())).willReturn(세트메뉴);
        
        // When
        MenuGroup actual = menuGroupService.create(세트메뉴);
        
        // Then
        assertAll(
                () -> assertNotNull(actual),
                () -> assertEquals(세트메뉴.getId(), actual.getId()),
                () -> assertEquals(세트메뉴.getName(), actual.getName())
        );
```